db/dependents/dependents_utils.py

```diff
@@ -139,6 +164,27 @@ def _get_table_dependents(pg_identify_object, base):
     return base.where(pg_identify_object.c.type == 'table')
 
 
+def _get_function_dependents(pg_depend, pg_identify_object, pg_proc):
+    return (
+        select(
+            pg_depend,
+            # the same as with pg_trigger table
+            collate(pg_proc.c.proname, 'default').label('objname'),
```

Comment:
I think you can call `collate` on `pg_identify_object.c.name` instead of joining `pg_proc.c.proname` and calling `collate` on it. This would enable us to use `_get_dependency_pairs_stmt` instead of a creating a similar `select` statement for `triggers` and `functions`

Reply:
The thing is `pg_identify_object` does not return names for `triggers` and `functions`. That is why I created separate methods with joins with their catalogues, to get that data from there. And that is also the reason for manually changing the collation. Because those catalogues have other collations